topics/reactphp/index.md

```diff
@@ -0,0 +1,11 @@
+---
+related: php
+created_by: Igor Wiedler
+display_name: ReactPHP
+github_url: https://github.com/reactphp
+released: April 14, 2012
+short_description: Event-driven, non-blocking I/O with PHP.
+topic: reactphp
+url: https://reactphp.org/
+---
+ReactPHP is a low-level library for event-driven programming in PHP. At its core is an event loop, on top of which it provides low-level utilities, such as: Streams abstraction, async dns resolver, network client/server, http client/server, interaction with processes.
```

Comment:
A few style requests:

- `Streams` => `a streams`
- `async` => `asynchronous`
- `dns` => `DNS`
- `http` => `HTTP`
- Add an "and" before `interaction`.